src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+
+  if (logo) {
+    logo.addEventListener('click', () => {
+      resolve();
+    });
+  }
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('error'));
+  }, 3000);
+});
+
+const successHandler = () => {
+  const message = document.createElement('div');
+
+  message.classList.add('message');
+  message.textContent = 'Promise was resolved!';
+  document.body.appendChild(message);
+};
```

Comment:
Consider separating the creation of the message from its insertion into the body. This way, the successHandler function only creates a message and returns it, and you could have another function that inserts messages into the body. This would make your code more modular and each function would have a single responsibility.